### PR TITLE
Specify log file in configuration file

### PR DESCRIPTION
**Pull request by dbellinghoven**
Allow users to specify directory to which log files will be written within the configuration file

### CODE REVIEW

**Comment by alexdulin** [comment]
### Concourse CI
All tests passing :white_check_mark:

**Comment by alexdulin** [comment]
### Concourse CI
All tests passing :white_check_mark:

**Comment by alexdulin** [comment]
### Concourse CI
All tests passing :white_check_mark:

**Comment by alexdulin** [comment]
### Concourse CI
All tests passing :white_check_mark:

**Comment by alexdulin** [comment]
### Concourse CI
Tests failing :x:

**Comment by alexdulin** [comment]
### Concourse CI
Tests failing :x:

**Comment by alexdulin** [comment]
### Concourse CI
All tests passing :white_check_mark:

**Comment by alexdulin** [comment]
### Concourse CI
All tests passing :white_check_mark:

File: ci/run-tests.sh [hunk 1]
```diff
@@ -34,9 +34,16 @@ echo "==> Fetching dependencies"
 
 dep ensure
 
+echo "==> Creating a new non-root user"
+
+readonly new_user="foobar"
+readonly new_group="foo"
```
Comment (alexdulin) [inline ci/run-tests.sh]: Do you want to make this **potato**?

Comment (dbellinghoven) [reply ci/run-tests.sh]: Absolutely

**Comment by alexdulin** [comment]
### Concourse CI
All tests passing :white_check_mark:

**Pull request merged**